<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapWorld</class>
 <widget class="QDialog" name="MapWorld">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Карта объектов</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/rec/img/ico.ico</normaloff>:/rec/img/ico.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QWidget" name="verticalWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>256</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(127, 127, 127, 255), stop:1 rgba(200, 200, 200, 255));</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="searchObjectLabel">
        <property name="styleSheet">
         <string notr="true">border-radius: 5px;
border-width: 2px;
border-radius: 10px;
font: 14px;
min-width: 10em;
padding: 3px;</string>
        </property>
        <property name="text">
         <string>Поиск объекта</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineSearch">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(238, 238, 236);
border-radius: 5px;
border-width: 2px;
border-radius: 10px;
font: 14px;
min-width: 10em;
padding: 6px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchBtn">
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(238, 238, 236);
border-radius: 5px;
border-width: 2px;
border-radius: 10px;
font: 14px;
min-width: 10em;
padding: 6px;</string>
        </property>
        <property name="text">
         <string>Поиск...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="informObjectLabel">
        <property name="styleSheet">
         <string notr="true">border-radius: 5px;
border-width: 2px;
border-radius: 10px;
font: 14px;
min-width: 10em;
padding: 3px;</string>
        </property>
        <property name="text">
         <string>Информация об объекте</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(238, 238, 236);
border-radius: 5px;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QQuickWidget" name="Map">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/rec/img/fonSetting.jpg);</string>
     </property>
     <property name="resizeMode">
      <enum>QQuickWidget::SizeRootObjectToView</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
